mindone/transformers/modeling_utils.py

```diff
@@ -2563,6 +2639,9 @@ def _find_mismatched_keys(
             # Whole checkpoint
             state_dict = _convert_state_dict(model, state_dict, prefix)
 
+            if key_renaming_mapping:
+                state_dict = {key_renaming_mapping[k]: v for k, v in state_dict.items() if k in key_renaming_mapping}
```

Comment:
consider disaggregating the name mapping function into two parts: 1) pt2hf, 2) hf2ms